### PR TITLE
Release/v0.10.0 apr 2024

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Release branch containing:**
- https://github.com/fivetran/dbt_netsuite_source/pull/49
- https://github.com/fivetran/dbt_netsuite_source/pull/47

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 packages:
-- package: fivetran/fivetran_utils
-  version: [">=0.4.0", "<0.5.0"]
+# - package: fivetran/fivetran_utils
+#   version: [">=0.4.0", "<0.5.0"]
+- git: https://github.com/fivetran/dbt_fivetran_utils.git
+  revision: feature/add-lookback-macro
```
Comment (fivetran-catfritz) [inline packages.yml:5]: Update before release
```suggestion
- package: fivetran/fivetran_utils
  version: [">=0.4.0", "<0.5.0"]
```

Comment (fivetran-catfritz) [reply packages.yml]: This was reverted.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: models/netsuite2/stg_netsuite2__transactions.sql [hunk 2]
```diff
@@ -22,6 +22,7 @@ final as (
     
     select
         _fivetran_synced,
+        cast(_fivetran_synced as date) as _fivetran_synced_date,
```
Comment (fivetran-jamie) [inline models/netsuite2/stg_netsuite2__transactions.sql:25]: should this include a `date_trunc`, or does casting as a `date` automatically remove the time-part of the timestamp?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+- Added column `_fivetran_synced_date` to model `stg_netsuite2__transactions` for use in downstream models.
+- To reduce storage, updated default materialization of staging models to views. 
+>  ⚠️ Running a `--full-refresh` will be required if you have previously run these staging models as tables and get the following error: 
+> ```
+> Trying to create view <model path> but it currently exists as a table. Either drop <model path> manually, or run dbt with `--full-refresh` and dbt will drop it for you.
+> ```
+
+## Under the Hood:
+- Added integration testing pipeline for Databricks SQL Warehouse.
+- Included auto-releaser GitHub Actions workflow to automate future releases.
```
Comment (fivetran-jamie) [inline CHANGELOG.md:13]: i don't actually see the auto-releaser workflow files in this PR/[branch](https://github.com/fivetran/dbt_netsuite_source/tree/release/v0.10.0-apr-2024/.github)

File: models/netsuite2/stg_netsuite2__transactions.sql (hunk 2 above)
Comment (fivetran-catfritz) [reply models/netsuite2/stg_netsuite2__transactions.sql:25]: The casting does remove the time-part, if you check out my validation from my ticket that shows the result of the cast. The date_trunc is the one that leaves the time-part I believe.

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:13]: 🤦‍♀️ Thank you. Added!

**Review by fivetran-catfritz — COMMENTED** [review]
Thank you!

**Review by fivetran-jamie — APPROVED** [review]
lgtm!

**Pull request merged**